pkgs/applications/science/misc/degrib/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, zlib
+, tcl
+, tcltls
+, tk
+}:
+
+stdenv.mkDerivation {
+  pname = "degrib";
+  version = "2.25";
+
+  src = fetchTarball {
+    url = "https://lamp.mdl.nws.noaa.gov/lamp/Data/degrib/download/archive/degrib-20200921.tar.gz";
+    sha256 = "1md33zbp3bs85d0v2i77zyq4clabhzqpwg17jy9rnyn40dc1kjc7";
+  };
+
+  buildInputs = [
```

Comment:
Isn't at least one of these nativeBuildInputs?

Reply:
Would `strictDeps = true;` settle this?